lib/ansible/modules/crypto/get_certificate.py

```diff
@@ -129,7 +133,7 @@ def main():
     module = AnsibleModule(
         argument_spec=dict(
             ca_certs=dict(required=False, type='path', default=None),
-            host=dict(required=True),
+            host=dict(required=True, type='str'),
```

Comment:
And we tend to start with the `type=`, since this is the only key that every parameter defines.
The order usually is: `type`, `required`, `default`, `choices`, ...